android/anc/src/main/java/org/smartregister/fhircore/anc/ui/report/ReportResultPage.kt

```diff
@@ -45,22 +48,34 @@ fun ReportResultPreview() {
   ReportResultPage(
     topBarTitle = "PageTitle",
     onBackPress = {},
-    reportMeasureItem = ReportItem(description = "Test Description"),
-    true,
+    reportMeasureItem =
+      ReportItem(
+        description = "Description For Preview, i.e 4+ Anc women etc, 2 lines text in preview"
```

Comment:
I guess I am to replace this after running measure reporting

Reply:
nops, you dont need to replace, its for preview of result page, excluded from coverage